sql/glam-fenix-dev/glam_etl/org_mozilla_fenix_glam_nightly__scalar_bucket_counts_v1/query.sql

```diff
@@ -89,39 +89,27 @@ RETURNS ARRAY<
   )
 );
 
-WITH
--- Cross join with the attribute combinations to reduce the query complexity
--- with respect to the number of operations. A table with n rows cross joined
--- with a combination of m attributes will generate a new table with n*m rows.
--- The glob ("*") symbol can be understood as selecting all of values belonging
--- to that group.
-static_combos AS (
+WITH build_ids AS (
   SELECT
-    combos.*
+    app_build_id,
+    channel,
   FROM
-    UNNEST(
-      ARRAY<STRUCT<ping_type STRING, os STRING, app_build_id STRING>>[
-        (NULL, NULL, NULL),
-        (NULL, NULL, "*"),
-        (NULL, "*", NULL),
-        ("*", NULL, NULL),
-        (NULL, "*", "*"),
-        ("*", NULL, "*"),
-        ("*", "*", NULL),
-        ("*", "*", "*")
-      ]
-    ) AS combos
+    glam_etl.org_mozilla_fenix_glam_nightly__clients_scalar_aggregates_v1
+  GROUP BY
+    1,
```

Comment:
it's usually good to write the column names explicitly to avoid bugs in case the selected columns get changed